public/js/main.js

```diff
@@ -111,57 +111,56 @@ async function runQueryByID(query) {
  * set queryParams as a json object containing the query parameters
  */
 function parseQueryParams() {
-    const url = location.search;
-    const query = url.substr(1);
-    const result = {global: {}, templateFields: {}};
-    query.split('&').forEach((param) => {
-        const item = param.split('=');
-        const parameter = getParameter(item);
-        if (parameter) {
-            if (parameter.type === 'global') {
-                result.global[decodeURIComponent(parameter.key)] = parameter.value;
-            } else if (parameter.type === 'templateField') {
-                result.templateFields[decodeURIComponent(parameter.key)] = parameter.value;
-            }
-        }
-    });
-    queryParams = result;
+  const result = {global: {}, templateFields: {}};
+  const parameters = new URLSearchParams(location.search);
```

Comment:
Do we have any concern related to the browser compatibility of URLSearchParams ? https://caniuse.com/urlsearchparams

I don't see incompatibility but just making sure that it is not working with IE and some old versions and if it is okay

Reply:
no we don't support EI and for others
last 3 Firefox major versions
last 3 Chrome major versions
last 3 ChromeAndroid major versions
last 1 Edge major versions
last 1 Safari major versions
last 1 iOS major versions
last 1 Opera major versions

so it should be fine. (we are also using it in the frontend)